boards/arm/nrf52_pca10040/nrf52_pca10040.dts

```diff
@@ -121,23 +148,20 @@ arduino_i2c: &i2c0 {
 &spi0 {
 	/* Cannot be used together with i2c0. */
 	/* status = "okay"; */
-	sck-pin = <27>;
-	mosi-pin = <26>;
-	miso-pin = <25>;
```

Comment:
Why remove all these assignments when you could just change the pin 25, which you need to release, to 28 for instance?

Reply:
My thought was that i2c0 is enabled and spi0 remains disabled so it didn't make sense to assign pins to a disabled SPI.  If someone wants to use that SPI they still have to add an overlay.

This also applies to i2c1 but doesn't have the above double assign issue that pin 25 (sck) had.

To be more consistent and simpler would it be better to not assign pins to both spi0 and i2c1 as they are disabled?  Anyone using a shield with the enabled spi1 (arduino_spi) and i2c0 (arduino_i2c) will not be affected and will have an easy path.  This also keeps the pin assignments close together.

Reply:
@anangl I've just read through how it's assigned in the nrf52840_pca10056.dts so it seems assigning pins to disabled peripherals is part of the current pattern (is this just giving the end user a starting point for the peripheral?) so I'll update with the changes you recommended.